test/syntax/idl/identifier-qualified-names.widl

```diff
@@ -26,3 +26,8 @@
       // Attribute identifier: "value"
       attribute DOMString? _value;
     };
+
+interface FooEventTarget {
+  // Argument names allow some selected keywords
+  void addEventListener(EventListener? callback);
```

Comment:
Does callback need the leading underscore?

Reply:
No, it's an allowed keyword in argument name position.